physlr/physlr.py

```diff
@@ -1201,6 +1207,27 @@ def physlr_overlap(self):
             (u, v) for bxs in progress(mxtobxs.values()) for u, v in itertools.combinations(bxs, 2))
         print(int(timeit.default_timer() - t0), "Loaded", len(edges), "edges", file=sys.stderr)
 
+        edge_list = list(edges)
```

Comment:
Couldn't you just iterate over the `edges` counter object instead of doing this explicit conversion?

Reply:
Yeah that is the local version. I wasn't sure all the intricacies would work out so I converted it to a list first. The local copy uses `edges`

Reply:
But you aren't updating `edges` variable until the very end, right? I don't quite understand what you're saying..

Reply:
By local, I meant the local copy of the code on my pc. I have implemented it like you suggested.